library/core/src/any.rs

```diff
@@ -605,7 +605,8 @@ impl dyn Any + Send + Sync {
 #[derive(Clone, Copy, Debug, Eq, PartialOrd, Ord)]
 #[stable(feature = "rust1", since = "1.0.0")]
 pub struct TypeId {
-    t: u128,
+    // See #115620 for this representation.
```

Comment:
```suggestion
    // We avoid using `u128` because that imposes higher alignment requirements on many platforms.
    // See issue #115620 for more information.
```